python/orion_finance_sdk_py/costs/dates.py

```
@@ -0,0 +1,28 @@
+def parse_cost_timestamp(timestamp: str | None) -> tuple[str, int]:
+    """Parse a manager-facing as-of date into ``(YYYY-MM-DD, unix_seconds)``.
+
+    ``None`` means now (UTC). A calendar date is the last second of that UTC
+    day so the snapshot is end-of-day state.
+    """
+    if timestamp is None:
+        now = datetime.now(timezone.utc)
+        return now.strftime(_DATE_FMT), int(now.timestamp())
+
+    try:
+        day = datetime.strptime(timestamp, _DATE_FMT).replace(tzinfo=timezone.utc)
```

Ruff (0.16.1)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)